sample/DokanNetMirror/Win32.cs

```diff
@@ -0,0 +1,12 @@
+﻿using System.Runtime.InteropServices;
+using Microsoft.Win32.SafeHandles;
+
+namespace DokanNetMirror
+{
+    public class Win32
```

Comment:
- Please add XML comments to the class and method. See [NativeMethods.cs](https://github.com/dokan-dev/dokan-dotnet/blob/master/DokanNet/Native/NativeMethods.cs) for inspiration.
- Please rename the class to `NativeMethods`, see [CA1060: Move P/Invokes to NativeMethods class](https://msdn.microsoft.com/en-us/library/ms182161(v=vs.110).aspx)